blog/_posts/2023-12-20-gsoc-report.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog-detail
+post-type: blog
+by: Anatolii Kmetiuk, Scala Center
+title: "Google Summer of Code 2023 Report"
+---
+We've just wrapped up another fruitful iteration of the Google Summer of Code initiative. This program is a key component of the Scala Center's approach to encouraging more newcomers to contribute to the Scala language, reinforcing its commitment to making Scala contributions more accessible to beginners.
+
+This time around, we had the pleasure of welcoming 6 contributors. Their contributions were directed toward various projects, including data visualization, type-level and asyncronous programming, metaprogramming and more. This article provides a brief overview of their achievements, along with links for further details.
+
+## Unlocking the Full Potential of Scala Web Development
+Akseli Ristola worked on integrating Scala 3 support into a [web development template](https://github.com/wiringbits/scala-webapp-template/commits?author=akseliristola) project under the supervision of Alexis Hernandez. This project focused on leveraging the advanced features of Scala 3 to enhance web application development, addressing current limitations and exploring new potentials in Scala-based web frameworks.
+
+Akseli successfully completed two major milestones in this project. The first involved developing a robust integration process for Scala 3, ensuring compatibility and efficiency. The second milestone was the implementation of new features and optimizations specific to Scala 3, which are expected to significantly improve the performance and scalability of web applications developed using Scala.
+
+## http4s Ember WebSocket Client
+In this project, Dang Trung Hieu worked under the mentorship of Arman Bilge, Christopher Davenport and Diego Alonso to implement a [WebSocket client](https://github.com/http4s/http4s/pull/7196) for the [http4s](https://github.com/http4s/http4s) Ember backend. The goal was to extend the capabilities of the http4s library, a Scala HTTP library, with robust WebSocket support to enable real-time communication features in web applications.
+
+The project's success lies in the development of a fully functional WebSocket client that integrates seamlessly with the existing http4s Ember framework. This integration enhances the library's utility in building modern, real-time web applications, thereby enriching the Scala ecosystem.
+
+## Data Visualization with Doodle
```

Comment:
@danielost, @noelwelsh, could you please review this section? Feel free to suggest changes, or rewrite it to your liking altogether!

Reply:
Looks good to me!

Reply:
Me too!